lib/actions/ssh.coffee

```diff
@@ -118,7 +118,8 @@ module.exports =
 				 root@#{options.deviceIp}"
 
 			if not options.host
-			 command += " docker exec -ti #{container} /bin/sh"
+				shellCmd = '/bin/sh -c $"\'if [ -e /bin/bash ]; then exec /bin/bash; else exec /bin/sh; fi\'"'
```

Comment:
@Page- fixed - this particular bash snippet still is kind of hard to read though :/